### PR TITLE
Swap HTML content from file to database storage

### PR DESCRIPTION
**Pull request by deshario**
- Add a new table to store HTML content
- Store the HTML content in the DB
- Query the HTML content from the DB and render it on the client

---

**Note**: This PR includes a migration

Please run **`yarn migrate`** before running the application

Reference: https://github.com/deshario/web-scraper/issues/26

### CODE REVIEW

File: apps/client/src/components/HtmlViewer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      })
+
+      tempDiv.remove()
+    }
+  }, [html])
+
+  return <div dangerouslySetInnerHTML={{ __html: html }} />
+}
+
+export default HTMLViewer
```
Comment (deshario) [inline apps/client/src/components/HtmlViewer/index.tsx:56]: Somehow, the `dangerouslySetInnerHTML` approach is not working well with inline scripts, and this issue is affecting the rendering of images within the DOM

To address this issue, I came up with injecting scripts, which seems to have resolved the issue.

I also attempted to use a different library that sanitizes HTML content, but the output appears to be different. It seems that the sanitization process removed a significant portion of the scripts, resulting in unexpected output

Comment (deshario) [inline apps/client/src/components/HtmlViewer/index.tsx:56]: Somehow, the `dangerouslySetInnerHTML` approach is not working well with inline scripts, and this issue is affecting the rendering of images within the DOM

To address this issue, I came up with injecting scripts, which seems to have resolved the issue.

I also attempted to use a different library that sanitizes HTML content, but the output appears to be different. It seems that the sanitization process removed a significant portion of the scripts, resulting in unexpected output

Comment (olivierobert) [reply apps/client/src/components/HtmlViewer/index.tsx:56]: Most of the issues can be solved by using an iframe and the `srcdoc` attribute (see: https://www.w3schools.com/tags/att_iframe_srcdoc.asp). While iframes are usually frowned upon due to accessibility issues, they should be the goto solution to render a page into another page, which is the use case at hand.

Comment (deshario) [reply apps/client/src/components/HtmlViewer/index.tsx:56]: That's really nice, Thanks

**Pull request merged**